bci_essentials/classification/erp_rg_classifier.py

```diff
@@ -190,7 +198,9 @@ def fit(
         preds = np.zeros(len(self.y))
 
         #
-        def erp_rg_kernel(X, y):
+        def erp_rg_kernel(
+            X: np.ndarray, y: np.ndarray
+        ) -> tuple[Generic_classifier, np.ndarray, float, float, float]:
```

Comment:
What is this line doing? Is it defining the output of the function?

Reply:
Yes, the "->" operator is used to define the return type of functions. Since only one object can be returned, for multiple return values (like this case) the type has to be some kind of array-like object that includes all the values

Reply:
Ah makes sense, thanks!